PasscodeLock/PasscodeLockViewController.swift

```diff
@@ -93,6 +93,14 @@ open class PasscodeLockViewController: UIViewController, PasscodeLockTypeDelegat
         self.updatePasscodeView()
         self.setupEvents()
     }
+	
+	open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+		super.traitCollectionDidChange(previousTraitCollection)
+
+		for (index, element) in self.placeholders.enumerated() {
+			self.animatePlacehodlerAtIndex(index, toState: element.currentState)
```

Comment:
Ok, but looks hacky. Is there another alternative ?

Reply:
I can try more. But I am afraid it's an iOS bug. The system care by itself once we provide both colors (light and dark). The bug is very inconsistent, sometimes it happens sometimes not

Reply:
If I'm not mistaken, the color switch is not automatic for `CALayer`s, which is what is used to determine the border color

Reply:
Yes, `CALayer` doesn't support dynamic colors, but even trying `myColor.resolvedColor(with: self.traitCollection).cgColor` didn't solve the problem. The problem is not only in the border sometimes the background is wrong too, as in the picture below
![IMG_EA669B6F7F0B-1](https://user-images.githubusercontent.com/4924719/107077648-f1f02000-67cb-11eb-913a-2defe099be66.jpeg)
: 

The first 2 circles should be blue too. But when I changed from dark to light mode, they just got wrong.

Reply:
Yes, `CALayer` doesn't support dynamic colors. Even trying `myColor.resolvedColor(with: self.traitCollection).cgColor` didn't solve the problem. BTW, the problem is not only with the border, sometimes the background color gets wrong too, like in the pic below, where the first 2 circles should be blue too, but something got wrong when changing from dark to light mode. :(

![IMG_EA669B6F7F0B-1](https://user-images.githubusercontent.com/4924719/107078309-e6e9bf80-67cc-11eb-87cd-525affe1a859.jpeg)